Docker

```diff
@@ -0,0 +1,23 @@
+# Use an official Node.js runtime as the base image
+FROM node:14
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy the package.json and package-lock.json files to the container
+COPY package*.json ./
+
+# Install project dependencies
+RUN npm install --global @gridsome/cli
+
+
+# Copy the rest of your application code to the container
+COPY . .
+
+RUN npm install
+
+# Expose the port that Gridsome uses (default is 8080)
```

Comment:
Please update the comment to accurately describe the function of the expose command.

"[...] The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container [...]
> References https://docs.docker.com/engine/reference/builder/#expose

Reply:
Thank you for your response @dpliakos . I concur with your assessment that Docker may not be the most practical choice for development purposes. However, I suggested the utilization of Docker primarily due to its ability to offer a seamless and consistent environment across various machines, alleviating concerns related to npm or other technology dependencies. In my view, incorporating Docker, or alternatively, a Docker Compose file within our repository aligns with established best practices observed in many GitHub repositories.